src/toHaveStyle.js

```diff
@@ -1,9 +1,9 @@
-import { parse } from 'css';
 import { checkHtmlElement, getTag, InvalidCSSError } from './utils';
 import { printSecSuccess, printSuccess, printSecError, printError } from './printers';
+import cssParse from 'css/lib/parse';
```

Comment:
I guess that should be on the first line in this file, since it is module dependency, then own dependencies goes